documentation/src/main/asciidoc/stories/assembly_wildfly_modules.adoc

```diff
@@ -16,7 +16,7 @@ endif::community[]
 //Downstream content
 ifdef::downstream[]
 In Red Hat JBoss EAP (EAP) 7.4 applications can directly handle the `infinispan` subsystem without the need to separately install {ProductName} modules.
-After EAP 7.4 GA is released, {ProductName} will no longer provide EAP modules for download.
+However, your deployment requires the EAP modules to use advanced capabilities such as indexing and querying.
```

Comment:
I'm not sure if we should make any statements or promises about future versions. The guidance from CCS is to avoid such statements (except deprecation notices).